### PR TITLE
Zoom.ZoomOutlookPlugin_Manual version 5.13.5

### PR DESCRIPTION
**Pull request by Simonhua9**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/94140)

### CODE REVIEW

File: manifests/z/Zoom/ZoomOutlookPlugin_Manual/5.13.5/Zoom.ZoomOutlookPlugin_Manual.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Zoom.ZoomOutlookPlugin_Manual
```
Comment (stephengillie) [inline manifests/z/Zoom/ZoomOutlookPlugin_Manual/5.13.5/Zoom.ZoomOutlookPlugin_Manual.installer.yaml:4]:
```suggestion
PackageIdentifier: Zoom.ZoomOutlookPlugin
```

Comment (stephengillie) [reply manifests/z/Zoom/ZoomOutlookPlugin_Manual/5.13.5/Zoom.ZoomOutlookPlugin_Manual.installer.yaml:4]: Please update the file path to match, when updating the PackgeIdentifier.

File: manifests/z/Zoom/ZoomOutlookPlugin_Manual/5.13.5/Zoom.ZoomOutlookPlugin_Manual.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Zoom.ZoomOutlookPlugin_Manual
```
Comment (stephengillie) [inline manifests/z/Zoom/ZoomOutlookPlugin_Manual/5.13.5/Zoom.ZoomOutlookPlugin_Manual.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: Zoom.ZoomOutlookPlugin
```

File: manifests/z/Zoom/ZoomOutlookPlugin_Manual/5.13.5/Zoom.ZoomOutlookPlugin_Manual.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: Zoom.ZoomOutlookPlugin_Manual
```
Comment (stephengillie) [inline manifests/z/Zoom/ZoomOutlookPlugin_Manual/5.13.5/Zoom.ZoomOutlookPlugin_Manual.yaml:4]:
```suggestion
PackageIdentifier: Zoom.ZoomOutlookPlugin
```

**Comment by stephengillie** [comment]
Hi @Simonhua9,

Thanks for submitting an updated version of this package. The suggested PackageIdentifier changes will help it fit alongside the previous version from #84733.

**Comment by stephengillie** [comment]
Hi @Simonhua9,

Validation failed because the file path doesn't match the PackageIdentifier:
>  Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 'manifests\z\Zoom\ZoomOutlookPlugin_Manual\5.13.5', expected 'manifests\z\Zoom\ZoomOutlookPlugin\5.13.5'

Please update the file names and paths, from 
```
..../manifests/z/Zoom/ZoomOutlookPlugin_Manual/5.13.5/Zoom.ZoomOutlookPlugin_Manual...
```
to 
```
..../manifests/z/Zoom/ZoomOutlookPlugin/5.13.5/Zoom.ZoomOutlookPlugin...
```
as part of updating the PackageIdentifier.

**Comment by Simonhua9** [comment]
> Hi @Simonhua9,
> 
> Validation failed because the file path doesn't match the PackageIdentifier:
> 
> > Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 'manifests\z\Zoom\ZoomOutlookPlugin_Manual\5.13.5', expected 'manifests\z\Zoom\ZoomOutlookPlugin\5.13.5'
> 
> Please update the file names and paths, from
> 
> ```
> ..../manifests/z/Zoom/ZoomOutlookPlugin_Manual/5.13.5/Zoom.ZoomOutlookPlugin_Manual...
> ```
> 
> to
> 
> ```
> ..../manifests/z/Zoom/ZoomOutlookPlugin/5.13.5/Zoom.ZoomOutlookPlugin...
> ```
> 
> as part of updating the PackageIdentifier.

Hi @stephengillie,

Thank you for checking. I noticed Zoom Outlook Plugin does not have the new version 5.13, so I submitted this pull request. 

When I used the wingetcreate to create YAML file, I could not use the name ..../manifests/z/Zoom/ZoomOutlookPlugin/5.13.5/Zoom.ZoomOutlookPlugin so I've added a suffix there. Can you please help advise how I can make the change there?

Thank you!

Best regards,
Simon Hua

**Comment by stephengillie** [comment]
Hi @Simonhua9,

Why wouldn't wingetcreate allow you to use the name? Did it give an error?

**Comment by ItzLevvie** [comment]
Close with reason: Stale pull request;

**Pull request closed without merging**